ddm_trainer.py

```diff
@@ -53,37 +56,26 @@ def main(cfg: DictConfig) -> None:
 
     model = Model()
     X, y = model.load_csv(
```

Comment:
Do we not need to do `load_csv` anymore, since you have methods for `get_train_set` and `get_test_set`?